src/frontend/csrs-portal/src/app/components/landing/landing.component.scss

```diff
@@ -0,0 +1,589 @@
+@import "palette";
+@import "bootstrap/bootstrap-imports";
+
+h1 {
+  font-size: 3em;
+  line-height: 1.1em;
+  margin-bottom: 50px;
+  font-weight: bold;
+  text-shadow: 2px 2px 4px #000;
```

Comment:
Can you remove box shadow from h1 styling, box shadow is blurring the text

Reply:
It was removed.

Reply:
Talked to Kartheek. As per my understanding he will tackle remaining cleanup as part of a new cleanup and refactoring ticket he is working on.